public/app/features/explore/LogsContainer.tsx

```diff
@@ -90,6 +90,16 @@ export class LogsContainer extends PureComponent<LogsContainerProps> {
     return [];
   };
 
+  showContextToggle = (): boolean => {
```

Comment:
Also, would you mind adding a comment here explaining (or pointing to some external resource that explains) why this is supported only by ES 5.0+ ? 🙏

Reply:
The reason I had to change the way `LogsContainer` was passing (and eventually calling) `showContextToggle` was because it was passing just the function which is not bound to the `datasourceInstance` object. That meant that the `this.esVersion` would fail because `this` evaluated to `undefined`. At some point the same change was made for `getLogRowContext` and this just keeps the two consistent.

I will add the comment.